frame/uniques/src/lib.rs

```diff
@@ -162,7 +162,7 @@ pub mod pallet {
 	}
 
 	#[pallet::storage]
-	#[pallet::storage_prefix = "Class"]
+	#[pallet::storage_prefix = "Collection"]
```

Comment:
We only set the `storage_prefix` to have the better name `Collection` inside the rust implementation without requiring a migration. Your proposed changes here require a migration.

Reply:
I created this PR because the current naming is not consistent.  In one interface (extrinsics) it was refereed to as Collections and items and in the rpc interface there were suddenly called Classes and instances. I only discovered this when I saw the mismatch in documentation of #13322

I think the changes made in #11389 should have renamed these interfaces as well.

**Pros:** It gives a better experience for the users (developers) where it is easy to understand the connection 
**Cons**: It is a change and it will break things.

Reply:
> does this macro also declare the storage name?

Yes. `storage_prefix` in this case is the storage name.

Reply:
@bkchr Is it possible to have meaningful names in the external API which out chaining the internal storage

Reply:
Currently not.